docs/UsersGuide/source/BuildingRunningTesting/RunSRW.rst

```diff
@@ -47,53 +47,62 @@ Download and Stage the Data
-.. table:: Data Locations for Level 1 Systems
-
-   +--------------+------------------------------------------------------------------------------+
-   | Machine      | File location                                                                |
-   +==============+==============================================================================+
-   | Cheyenne     | /glade/work/epicufsrt/contrib/UFS_SRW_data/develop/input_model_data/         |
-   +--------------+------------------------------------------------------------------------------+
-   | Gaea         | /lustre/f2/dev/role.epic/contrib/UFS_SRW_data/develop/input_model_data/      |
-   +--------------+------------------------------------------------------------------------------+
-   | Hera         | /scratch1/NCEPDEV/nems/role.epic/UFS_SRW_data/develop/input_model_data/      |
-   +--------------+------------------------------------------------------------------------------+
-   | Jet          | /mnt/lfs4/HFIP/hfv3gfs/role.epic/UFS_SRW_data/develop/input_model_data/      |
-   +--------------+------------------------------------------------------------------------------+
-   | NOAA Cloud   | /contrib/EPIC/UFS_SRW_data/develop/input_model_data/                         |
-   +--------------+------------------------------------------------------------------------------+
-   | Orion        | /work/noaa/epic-ps/role-epic-ps/UFS_SRW_data/develop/input_model_data/       |
-   +--------------+------------------------------------------------------------------------------+
-   | WCOSS2       | /lfs/h2/emc/lam/noscrub/UFS_SRW_App/develop/input_model_data/                |
-   +--------------+------------------------------------------------------------------------------+ 
-    
-For Level 2-4 systems, the data must be added to the user's system. Detailed instructions on how to add the data can be found in :numref:`Section %s: Downloading and Staging Input Data <DownloadingStagingInput>`. Sections :numref:`%s <Input>` and :numref:`%s <OutputFiles>` contain useful background information on the input and output files used in the SRW App. 
+.. list-table:: Data Locations for Level 1 Systems
+   :widths: 20 50
+   :header-rows: 1
+
+   * - Machine
+     - File location
+   * - Derecho
+     - /glade/work/epicufsrt/contrib/UFS_SRW_data/|data|/input_model_data
+   * - Gaea (C3/C4/C5)
+     - /lustre/f2/dev/role.epic/contrib/UFS_SRW_data/|data|/input_model_data/
+   * - Hera
+     - /scratch1/NCEPDEV/nems/role.epic/UFS_SRW_data/|data|/input_model_data/
+   * - Hercules
+     - /work/noaa/epic/role-epic/contrib/UFS_SRW_data/|data|/input_model_data/
+   * - Jet
+     - /mnt/lfs4/HFIP/hfv3gfs/role.epic/UFS_SRW_data/|data|/input_model_data/
+   * - NOAA Cloud
+     - /contrib/EPIC/UFS_SRW_data/|data|/input_model_data/
+   * - Orion
+     - /work/noaa/epic/role-epic/contrib/UFS_SRW_data/|data|/input_model_data/
+   * - WCOSS2
+     - /lfs/h2/emc/lam/noscrub/UFS_SRW_App/develop/input_model_data/
```

Comment:
@MichaelLueken @EdwardSnyder-NOAA Does the WCOSS location need to be updated to `UFS_SRW_data`?

Reply:
@MichaelLueken, I'll check out and test this release branch on WCOSS2, and let you know soon.

Reply:
Many thanks to @MatthewPyle-NOAA for providing the following:
```
Looks like it still has it there on WCOSS

ls /lfs/h2/emc/lam/noscrub/UFS_SRW_App/develop/input_model_data/
FV3GFS  GSMGFS  HRRR  NAM  RAP
```
So, `/lfs/h2/emc/lam/noscrub/UFS_SRW_App/develop/input_model_data/` is still the correct location on WCOSS2.

Reply:
@MichaelLueken, one we2e test has been completed successfully on WCOSS2 as follows:
```
       CYCLE                    TASK                       JOBID               STATE         EXIT STATUS     TRIES      DURATION
================================================================================================================================
201907010000               make_grid                   107737134           SUCCEEDED                   0         1          15.0
201907010000               make_orog                   107737172           SUCCEEDED                   0         1          42.0
201907010000          make_sfc_climo                   107737203           SUCCEEDED                   0         1          68.0
201907010000           get_extrn_ics                   107737135           SUCCEEDED                   0         1          11.0
201907010000          get_extrn_lbcs                   107737136           SUCCEEDED                   0         1          11.0
201907010000         make_ics_mem000                   107737305           SUCCEEDED                   0         1         127.0
201907010000        make_lbcs_mem000                   107737306           SUCCEEDED                   0         1         210.0
201907010000         run_fcst_mem000                   107737514           SUCCEEDED                   0         1         237.0


Summary of workflow status:
~~~~~~~~~~~~~~~~~~~~~~~~~~

  1 out of 1 cycles completed.
  Workflow status:  SUCCESS

========================================================================
End of output from script "launch_FV3LAM_wflow.sh".
========================================================================


The end-to-end run of the workflow for the forecast experiment specified
by expt_name has completed with the following workflow status (wflow_status):
  expt_name = "grid_RRFS_CONUS_25km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_v15p2"
  wflow_status = "SUCCESS"
```
As @MatthewPyle-NOAA mentioned above, the path to the input data is correct.

Reply:
Thank you very much for the confirmation that the WE2E test works on WCOSS2 and that the path to the input data is correct, @chan-hoo!  Resolving this conversation now.